atc/api/config_test.go

```diff
@@ -908,30 +908,16 @@ jobs:
 						request.Body = gbytes.BufferWithBytes(remoraPayload)
 					})
 
-					It("returns 200", func() {
-						Expect(response.StatusCode).To(Equal(http.StatusOK))
+					It("returns 400", func() {
```

Comment:
@vito  Your `UnmarshalConfig` refactor seems a breaking change. Originally, unknown fields at top level are silently swallowed, but now error will be thrown to client. I think which should be fine, but still want to confirm with you.